README.md

```diff
@@ -1,73 +1,69 @@
 # Project Name
 
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
+> We have created a command line TicTacToe Project using Ruby.
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
-
-## Live Demo
+- Ruby,
+- Vs.code,
+- Git and Github
 
-[Live Demo Link](https://livedemo.com)
+## Getting Started
 
+- clone this repository `https://github.com/rajkumardas2701/Tic-Tac-Toe`
+- cd to the folder `Tic-Tac-Toe`
+- To run the application use this command `ruby bin/main.rb`
 
-## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+### Rules of the game
 
+1. The game can only be played by 2 players (Player X and Player O).
 
-To get a local copy up and running follow these simple example steps.
+2. Player X begins the game.
 
-### Prerequisites
+3. Each player can only take one position at a time.
 
-### Setup
+4. Each player can only pick a position between 1 - 9.
 
-### Install
+5. Each player can only pick a position that has not been picked previously.
 
-### Usage
+6. A player wins if three cells in the board match either horizontally, vertically or diagonally.
 
-### Run tests
+7. The game comes to a draw if all cells in the board have been exhausted and there still is no winner.
 
-### Deployment
 
 
 
 ## Authors
 
-👤 **Author1**
+👤 **Evanson Igiri**
 
-- Github: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- Linkedin: [linkedin](https://linkedin.com/linkedinhandle)
+- Github: [@githubhandle](https://github.com/evansinho)
+- Twitter: [@twitterhandle](https://twitter.com/iamsinho1304)
+- Linkedin: [linkedin](https://linkedin.com/in/evanson-igiri)
 
-👤 **Author2**
+👤 **Rajkumar Das**
 
-- Github: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- Linkedin: [linkedin](https://linkedin.com/linkedinhandle)
+- Github: [@githubhandle](https://github.com/rajkumardas2701)
+- Twitter: [@twitterhandle](https://twitter.com/Rajkuma58621299)
+- Linkedin: [linkedin](https://www.linkedin.com/in/rajkumar-das-41308961/)
 
 ## 🤝 Contributing
 
 Contributions, issues and feature requests are welcome!
 
-Feel free to check the [issues page](issues/).
+Feel free to check the [issues page](https://github.com/rajkumardas2701/Tic-Tac-Toe/issues).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Odin Project
+- Ferguson Iyara
 
 ## 📝 License
```

Comment:
- [ ] If you won't include the license type it's better to just exclude the license header.

Reply:
A licence type is present